docs/conf.py

```diff
@@ -84,7 +85,23 @@
     "exclude-members": "__repr__,__weakref__,__metaclass__,__init__,model_config,model_fields,model_post_init"
 }
 
+# -- Doctest configuration -------------------------------------------------
+# Doctest global imports and instances
+import doctest
 
+doctest_default_flags = (
+    0
+    | doctest.DONT_ACCEPT_TRUE_FOR_1
+    | doctest.ELLIPSIS
+    | doctest.IGNORE_EXCEPTION_DETAIL
+    | doctest.NORMALIZE_WHITESPACE
+)
+
+doctest_global_setup = '''
+from ape import accounts
```

Comment:
we likely want to import all the ape top-level stuff here.

Reply:
`from ape import *`?

Reply:
that works!